src/api/spec/controllers/status/required_checks_controller_spec.rb

```diff
@@ -60,6 +63,15 @@
       it { expect(response).to have_http_status(:success) }
```

Comment:
My suspicion is that it is somehow this line. No idea what `it` is. It is not the `post` from above.

Reply:
The request is missing here.